packages/flutter_tools/test/commands.shard/hermetic/config_test.dart

```diff
@@ -288,7 +286,7 @@ class FakeAndroidSdk extends Fake implements AndroidSdk {
 
 class FakeFlutterVersion extends Fake implements FlutterVersion {
   @override
-  String channel;
+  late String channel;
```

Comment:
I tried making this a getter backed by a nullable private, but the analyzer insisted I just use `late`.